test/README.md

```diff
@@ -0,0 +1,215 @@
+# zAppBuild/test
+Test folder is designed to help test samples like the Mortgage Application against ZAppBuild.
+
+## Repository Legend
+Folder/File | Description | Documentation Link
+--- | --- | ---
+applications/MortgageApplication | This folder contains modified language scripts used to execute impact build by replacing these modified files with the original language files | [MortgageApplication](applications/MortgageApplication/README.md)
+test.groovy  | This is the main build script that is called to start the test process | [test.groovy](/test/README.md#testing-applications-with-zappbuild)
+testScripts  | This folder contains test scripts to execute full and impact builds | [testScripts](/test/testScripts/README.md)
+
+# Testing Applications with zAppBuild
+The main script for testing applications against zAppBuild is `test.groovy`. It takes most of its input from the command line to run full and impact builds. `test.groovy` once executed from the command line calls [fullBuild.groovy](/test/testScripts/fullBuild.groovy) and [impactBuild.groovy](/test/testScripts/impactBuild.groovy) scripts to perform an end to end test on the given feature branch with the program specified for impact build. 
+
+test.groovy script has five required arguments that must be present during each invocation:
+* --branch <arg> - zAppBuild branch to test
+* --app <arg> - Application that is being tested (example: MortgageApplication)
+* --url <arg> - DBB Web Application server URL
+* --hlq <arg> - HLQ for dataset reation / deletion (example: USER.BUILD)
+* --id <arg> - DBB Web Application user id
+
+test.groovy script has three optional argument that can be present during each invocation
+* --pw <arg> - DBB Web Application user password
+* --pwFile <arg> - DBB Web Application user password file
+* --verbose <arg> - Flag indicating to print trace statements
+* --propFiles <arg> - Absolute path to the location of the datasets.properties
+* --outDir <arg> - Absolute path to out directory
+
+# Examples of running an end to end test:
+
+```
+$DBB_HOME/bin/groovyz ${repoPath}/test/test.groovy -b testBranch -a MortgageApplication -q USER.BUILD -u urlToDbbWebApp -i userID -p pwd
+``` 
+```
+$DBB_HOME/bin/groovyz ${repoPath}/test/test.groovy -b testBranch -a MortgageApplication -q USER.BUILD -u urlToDbbWebApp -i userID -p pwd --propFiles /pathToDatasets/datasets.properties --outDir /pathToOutDir/out
```

Comment:
Please add a brief description, that with the invocation any kind of build properties are passed to zAppBuild

Reply:
Added "For this invocation of the test framework, it is assumed to have the dataset.properties defined to the actual execution environment" to test README documentation.